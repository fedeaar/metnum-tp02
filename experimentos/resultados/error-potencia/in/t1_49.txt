-4.842789557445975106e+02 -1.365900278068798457e+01 3.110992745385949831e+01 -8.734272173716581378e+00 -1.864679500758640174e+01 1.105993376096494671e+02 -1.430552454251085237e+01 1.398385220999873582e+02 4.117999142503142451e+01 5.721608769786621451e+01 -1.014068233604254488e+02 1.090647329631073461e+02 8.256700600939529977e+01 1.453917587084438523e+00 2.285326503018185207e+01 -4.558079152822053004e+00 -2.944614216871669132e+00 4.069804632578941295e+01 2.828188955224728929e+01 -4.843357431666338897e+01
-1.365900278068798457e+01 5.918363985394553310e+02 -1.249921652782975512e+00 -5.615892415473634713e+00 -3.668932460649958216e+01 -6.884779749692600426e+00 -1.614556290424327756e+01 -1.697364946171671463e+01 -4.107071542830408362e+01 -3.574739042767585318e+01 -6.452952470417822894e+01 -5.079855320149623488e+00 -2.302955732471159322e+01 -8.855861792946736477e-01 -2.881235293470303915e+01 -1.571429937278932343e+01 -1.549303184539603961e+01 -1.166992864832315746e+01 -2.424532581750140636e+00 -3.147352803513032882e+01
3.110992745385949476e+01 -1.249921652782976400e+00 -8.786146529698215772e+02 -1.286645322326616547e+00 5.159068491893625819e+00 4.125757112358980550e+01 1.164031760892765832e-01 5.481149514346279972e+01 2.768698554176525306e+01 3.164563365952341556e+01 -1.515339969378007901e+01 4.013796690119298916e+01 3.652662917479737814e+01 7.968522159396329574e-01 1.729196807369817535e+01 3.420300255276179868e+00 3.919210322910227973e+00 1.810624327358463148e+01 1.076270204807954123e+01 -7.028451268221203030e+00
-8.734272173716577825e+00 -5.615892415473634713e+00 -1.286645322326616103e+00 8.162124551258684733e+02 -1.898566572137567121e+01 -5.862410469749165465e+00 -8.231916864385480537e+00 -1.177579470092119607e+01 -2.250364311225751734e+01 -2.001766395717021396e+01 -3.200926774735524560e+01 -4.878993794323730704e+00 -1.381723923065167092e+01 -4.966418261094270870e-01 -1.566533354005806089e+01 -8.200799424889224198e+00 -8.116553788344925024e+00 -6.978694282590497622e+00 -1.849503937273139886e+00 -1.563283789687107372e+01
-1.864679500758641240e+01 -3.668932460649958216e+01 5.159068491893625819e+00 -1.898566572137567121e+01 1.598711728072699430e+02 1.116617780058562026e+01 -5.151550744990142050e+01 -1.018469520789351712e+01 -1.090202404134708019e+02 -8.882132973996843361e+01 -2.184520107282543790e+02 1.603537578536767327e+01 -4.424434426006518351e+01 -2.190053972634487867e+00 -7.819467932805252985e+01 -4.748046857287746292e+01 -4.637203912965356523e+01 -2.274440097711194397e+01 9.037985904216438993e-01 -1.062560129816143331e+02
1.105993376096494671e+02 -6.884779749692596873e+00 4.125757112358980550e+01 -5.862410469749167241e+00 1.116617780058561848e+01 -6.689840906113065557e+02 -2.940676149620767177e+00 1.963392003955271434e+02 9.240384002046569378e+01 1.079490631315952243e+02 -6.870839075904390825e+01 1.453093261697878518e+02 1.283989486407042193e+02 2.721241772023391903e+00 5.705207422155282870e+01 9.197282948707668382e+00 1.106273860806739329e+01 6.359435690502456850e+01 3.874219913653070790e+01 -3.219002479979472753e+01
-1.430552454251085237e+01 -1.614556290424327756e+01 1.164031760892774714e-01 -8.231916864385480537e+00 -5.151550744990143471e+01 -2.940676149620770730e+00 4.059704131457490917e+02 -1.508050449894963485e+01 -5.400928455972285747e+01 -4.574894429532309204e+01 -9.465063939968571560e+01 -5.509248672031190619e-01 -2.677839670344668122e+01 -1.131208763100488790e+00 -3.824496682129649372e+01 -2.186234615773089018e+01 -2.146317785526570887e+01 -1.363690951715768307e+01 -1.664392708724232284e+00 -4.610424344795907814e+01
1.398385220999873582e+02 -1.697364946171671463e+01 5.481149514346279972e+01 -1.177579470092119607e+01 -1.018469520789352245e+01 1.963392003955271718e+02 -1.508050449894964196e+01 -4.147479443658145328e+02 9.641949538340348624e+01 1.210596333285584052e+02 -1.371265903717000754e+02 1.925799121827158444e+02 1.574073690244947556e+02 3.062446162472544753e+00 5.715385814557159705e+01 1.594930582920929929e+00 4.264413420370764207e+00 7.778037211508636517e+01 5.060888020560494027e+01 -6.509844633970305949e+01
4.117999142503142451e+01 -4.107071542830408362e+01 2.768698554176525306e+01 -2.250364311225751734e+01 -1.090202404134707876e+02 9.240384002046569378e+01 -5.400928455972285747e+01 9.641949538340348624e+01 -1.806584058578248744e+02 -3.165435832214643241e+01 -2.596127235803491544e+02 9.534171949289199688e+01 2.481848872105079451e+01 -7.481053651085072254e-01 -4.852403156243434523e+01 -4.330283572725191021e+01 -4.116061406566167591e+01 1.144711266073740674e+01 2.198982153205437129e+01 -1.256077958861112336e+02
5.721608769786621451e+01 -3.574739042767585318e+01 3.164563365952341556e+01 -2.001766395717021041e+01 -8.882132973996843361e+01 1.079490631315951958e+02 -4.574894429532309204e+01 1.210596333285584336e+02 -3.165435832214642531e+01 -2.222076376421668158e+02 -2.311552410834201225e+02 1.096406531967539877e+02 4.720938730658259175e+01 -6.439732829278088655e-02 -2.879863951382925791e+01 -3.429828273018900120e+01 -3.212323747301504540e+01 2.267578569717165493e+01 2.636528617765359073e+01 -1.116222416423547941e+02
-1.014068233604254488e+02 -6.452952470417821473e+01 -1.515339969378007190e+01 -3.200926774735524560e+01 -2.184520107282544359e+02 -6.870839075904389404e+01 -9.465063939968571560e+01 -1.371265903717001038e+02 -2.596127235803491544e+02 -2.311552410834201510e+02 4.634508267283644045e+02 -5.736608555929433351e+01 -1.600198717872955569e+02 -5.735373561828401634e+00 -1.806600554809226651e+02 -9.439736687939041815e+01 -9.344452812491614679e+01 -8.081065542273657343e+01 -2.160520612631146520e+01 -1.795171811830178967e+02
1.090647329631073319e+02 -5.079855320149623488e+00 4.013796690119298916e+01 -4.878993794323729816e+00 1.603537578536767327e+01 1.453093261697878233e+02 -5.509248672031112903e-01 1.925799121827158444e+02 9.534171949289199688e+01 1.096406531967539877e+02 -5.736608555929433351e+01 -7.265375080728913417e+02 1.276378808119518027e+02 2.761671167905295210e+00 5.935749688999399609e+01 1.114396274578364299e+01 1.291927130596826423e+01 6.325492799367832220e+01 3.786882692112737203e+01 -2.669951097519248151e+01
8.256700600939534240e+01 -2.302955732471158612e+01 3.652662917479737814e+01 -1.381723923065167092e+01 -4.424434426006516219e+01 1.283989486407042193e+02 -2.677839670344668122e+01 1.574073690244947841e+02 2.481848872105079806e+01 4.720938730658260596e+01 -1.600198717872955001e+02 1.276378808119518027e+02 -3.488250803615501923e+02 1.213243369692500684e+00 1.017948675767041777e+01 -1.484250752356259184e+01 -1.277731642506447329e+01 4.179339873783938941e+01 3.243710907157333168e+01 -7.681807301781816477e+01
1.453917587084438745e+00 -8.855861792946736477e-01 7.968522159396329574e-01 -4.966418261094269759e-01 -2.190053972634488755e+00 2.721241772023391903e+00 -1.131208763100488790e+00 3.062446162472544753e+00 -7.481053651085072254e-01 -6.439732829278085879e-02 -5.735373561828400746e+00 2.761671167905295654e+00 1.213243369692500684e+00 -2.240005939650300206e+02 -6.905107659301226164e-01 -8.438978940546125740e-01 -7.894846238787563975e-01 5.834533010619683546e-01 6.654909038501768404e-01 -2.769184684601336155e+00
2.285326503018184141e+01 -2.881235293470303915e+01 1.729196807369817179e+01 -1.566533354005806444e+01 -7.819467932805255828e+01 5.705207422155282870e+01 -3.824496682129649372e+01 5.715385814557159705e+01 -4.852403156243435944e+01 -2.879863951382925791e+01 -1.806600554809226367e+02 5.935749688999401030e+01 1.017948675767041777e+01 -6.905107659301226164e-01 -1.178353155524862501e+02 -3.133610564219745953e+01 -2.992095083651248899e+01 4.440612568036202745e+00 1.338605607836342060e+01 -8.746960788148128074e+01
-4.558079152822054780e+00 -1.571429937278932343e+01 3.420300255276180756e+00 -8.200799424889222422e+00 -4.748046857287746292e+01 9.197282948707671935e+00 -2.186234615773089018e+01 1.594930582920929929e+00 -4.330283572725191021e+01 -3.429828273018900120e+01 -9.439736687939040394e+01 1.114396274578365009e+01 -1.484250752356259895e+01 -8.438978940546124630e-01 -3.133610564219745953e+01 1.772078641347125654e+02 -1.926758365273585838e+01 -7.702433126246619466e+00 1.546148677859274345e+00 -4.587833125831758707e+01
-2.944614216871663803e+00 -1.549303184539603961e+01 3.919210322910228417e+00 -8.116553788344925024e+00 -4.637203912965357944e+01 1.106273860806739329e+01 -2.146317785526570532e+01 4.264413420370765095e+00 -4.116061406566167591e+01 -3.212323747301504540e+01 -9.344452812491611837e+01 1.291927130596827311e+01 -1.277731642506447685e+01 -7.894846238787562864e-01 -2.992095083651248544e+01 -1.926758365273585838e+01 1.462720883007013413e+02 -6.672509521832187041e+00 2.048135550252426995e+00 -4.539868471076151479e+01
4.069804632578941295e+01 -1.166992864832315391e+01 1.810624327358463148e+01 -6.978694282590497622e+00 -2.274440097711194753e+01 6.359435690502456850e+01 -1.363690951715768307e+01 7.778037211508637938e+01 1.144711266073739253e+01 2.267578569717165493e+01 -8.081065542273655922e+01 6.325492799367832930e+01 4.179339873783936099e+01 5.834533010619685767e-01 4.440612568036202745e+00 -7.702433126246623019e+00 -6.672509521832187041e+00 -4.055007940926388414e+02 1.605111174585785250e+01 -3.880397142764737595e+01
2.828188955224728929e+01 -2.424532581750138860e+00 1.076270204807954123e+01 -1.849503937273139886e+00 9.037985904216436772e-01 3.874219913653070790e+01 -1.664392708724232284e+00 5.060888020560494027e+01 2.198982153205437129e+01 2.636528617765359073e+01 -2.160520612631146520e+01 3.786882692112737203e+01 3.243710907157333878e+01 6.654909038501767293e-01 1.338605607836342060e+01 1.546148677859272569e+00 2.048135550252425219e+00 1.605111174585784894e+01 -7.419626043460310711e+02 -1.019069003560933417e+01
-4.843357431666338186e+01 -3.147352803513032882e+01 -7.028451268221203030e+00 -1.563283789687107195e+01 -1.062560129816143331e+02 -3.219002479979472042e+01 -4.610424344795907814e+01 -6.509844633970305949e+01 -1.256077958861112194e+02 -1.116222416423547941e+02 -1.795171811830178967e+02 -2.669951097519248151e+01 -7.681807301781816477e+01 -2.769184684601336155e+00 -8.746960788148128074e+01 -4.587833125831758707e+01 -4.539868471076151479e+01 -3.880397142764736884e+01 -1.019069003560933595e+01 7.193323648000377943e+02
