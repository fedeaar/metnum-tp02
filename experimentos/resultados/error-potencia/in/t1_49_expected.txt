-8.900000000000000000e+02
-8.680000000000000000e+02
8.310000000000000000e+02
8.190000000000000000e+02
-8.180000000000000000e+02
8.070000000000000000e+02
-7.520000000000000000e+02
-6.680000000000000000e+02
6.030000000000000000e+02
-5.600000000000000000e+02
-4.340000000000000000e+02
4.290000000000000000e+02
-4.260000000000000000e+02
2.730000000000000000e+02
-2.240000000000000000e+02
-2.180000000000000000e+02
1.970000000000000000e+02
1.650000000000000000e+02
-1.200000000000000000e+02
-8.000000000000000000e+01
